spec/lib/terraforming/resource/security_group_spec.rb

```diff
@@ -339,12 +346,14 @@ module Resource
                   "ingress.1849628954.to_port" => "65535",
                   "ingress.1849628954.protocol" => "tcp",
                   "ingress.1849628954.cidr_blocks.#" => "0",
+                  "ingress.1849628954.prefix_list_ids.#" => "0",
                   "ingress.1849628954.security_groups.#" => "0",
                   "ingress.1849628954.self" => "true",
                   "ingress.1446312017.from_port" => "22",
                   "ingress.1446312017.to_port" => "22",
                   "ingress.1446312017.protocol" => "tcp",
                   "ingress.1446312017.cidr_blocks.#" => "1",
+                  "ingress.1446312017.prefix_list_ids.#"=>"0",
```

Comment:
__[rubocop]__
* Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->